volplugin/handlers.go

```diff
@@ -168,6 +173,9 @@ func mount(master, host string) func(http.ResponseWriter, *http.Request) {
 			return
 		}
 
+		stopChan := addStopChan(uc.Request.Name)
+		go heartbeatMount(master, ttl, ut, stopChan)
```

Comment:
We could use rbd locks to accomplish this. I can put in a ticket, but it's really out of scope (and a large change).